polyfill/intersection-observer.js

```diff
@@ -965,6 +965,11 @@ function getParentNode(node) {
     return getFrameElement(node);
   }
 
+  // If the parent has element that is assigned through shadow root slot
+  if (parent && parent.assignedSlot) {
```

Comment:
This looks good, but it also means that the second `if (parent && parent.assignedSlot) {` at line 978 is no longer needed, right?

Reply:
Yeah, makes sense.